Automattic/wp-calypso
# Jetpack Onboarding: Update illustrations

This PR updates the illustrations in several of the steps, as suggested in p6TEKc-1Rd-p2 #comment-4283.

**Site title step - before**
![](https://cldup.com/-76pPyx-Nu.png)

**Site title step - after**
![](https://cldup.com/4djb8AqW1D.png)

**Business address step - before**
![](https://cldup.com/Plv4zbgFdH.png)

**Business address step - after**
![](https://cldup.com/VSxQKSZO0v.png)

**Stats step - before**
![](https://cldup.com/MJIvVzxaEQ.png)

**Stats step - after**
![](https://cldup.com/A_Q7zZnylY.png)

To test:
* Checkout this branch
* Pick a Jetpack site.
* Head to `/wp-admin/admin.php?page=jetpack&action=onboard&calypso_env=development` on the Jetpack site to start the onboarding flow.
* On the site title step, verify the illustration looks good.
* In the site type step, select "Business".
* Skip to the business address step and verify the illustration looks good.
* Skip to the stats step and verify the illustration looks good.
